todo_app/templates/index.html

```diff
@@ -1,55 +1,70 @@
+    <h1 class="display-4">Do Me</h1>
+    <p class="lead">Jackie's First Python Application</p>
+    <p class="text">{{ repository_description }}</p>
   </div>
 
   <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-        {% for item in items %}
+    <div class="col-sm" >
+      <h2>All Tasks</h2>
+        {% if task_count == 0 %}
+        <p class="text">There are no tasks, hooray!</p>
+        {% else %}
+        <p class="text">You have {{ task_count }} tasks:</p>
+        {% endif %}
+        {% for task in tasks %}
           <div class="card" style="width: 18rem;">
-            <!--<img src="https://via.placeholder.com/300" class="card-img-top" alt="A placeholder image">-->
+            <!-- <img src="/static/icons/add.png" class="card-img-top" alt="Task Image"> -->
             <div class="card-body">
-              <h5 class="card-title">[{{ item.id }}] {{ item.title }}</h5>
-              <p class="card-text">{{ item.status }}</p>
-            </div>
-            <div class="card-body">
-              {% if item.status != "Done" %}
-
-                <!--<a class="card-link" href="{{ url_for('mark_as_done', itemId=item.id) }}">Done</a>   -->
-                
-                <form action="/mark_as_done/{{item.id}}" method="POST" class="form-group">
-                    <button type="submit" class="btn btn-primary">Mark as Done</button>
+              <h5 class="card-title">{{ task.name }}</h5>
+              <p class="card-text">[{{ task.id }}]</p>
+              <p class="card-text">{{ task.status }}</p>
+              <div class="btn-group" role="group" aria-label="Task Actions">
+                {% if task.status == "To Do" %}                
+                  <form action="/change_status/{{task.id}}?status=Doing" method="POST" class="form-group">
+                      <button type="submit" class="btn btn-secondary">>> Doing</button>
+                  </form>
+                {% endif %}
+                {% if task.status == "Doing" %}                
+                  <form action="/change_status/{{task.id}}?status=Done" method="POST" class="form-group">
+                      <button type="submit" class="btn btn-secondary">>> Done</button>
+                  </form>
+                {% endif %}
+                <!--TODO How can I change the button colour to RED?-->
+                <form action="/delete/{{task.id}}" method="POST" class="form-group">
+                  <button type="submit" class="btn btn-secondary" >Delete</button>
                 </form>
-
-              {% endif %}
-
-              <!--<a class="card-link" href="{{ url_for('delete', itemId=item.id) }}">Delete</a> -->
-
-              <form action="/delete/{{item.id}}" method="POST" class="form-group">
-                <button type="submit" class="btn btn-primary">Delete</button>
-              </form>
-
+              </div>
+              <img src="/static/icons/add.png" class="card-img-top" alt="Add Task Image">
```

Comment:
You asked why one of your images is displaying badly. I assume you mean this one? But I don't know how you wanted this to display so I can't really give advice

The class "card-img-top" that you added is meant to be used like this: https://getbootstrap.com/docs/4.0/components/card/#example (the grey box at the top of the card)